lib/Sema/TypeCheckAttr.cpp

```diff
@@ -5789,7 +5789,8 @@ IndexSubset *DifferentiableAttributeTypeCheckRequest::evaluate(
 void AttributeChecker::visitDifferentiableAttr(DifferentiableAttr *attr) {
   // Call `getParameterIndices` to trigger
   // `DifferentiableAttributeTypeCheckRequest`.
-  (void)attr->getParameterIndices();
+  if (attr->getParameterIndices() == nullptr)
+    attr->setInvalid();
```

Comment:
Instead, should `DifferentiableAttributeTypeCheckRequest` be responsible for setting the attr as invalid? The request is already doing so in some places when it calls `typecheckDifferentiableAttrforDecl`.

Reply:
Sure. I'll make the change.